android/ci_build_libs_cmake.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/sh
+set -e
+
+PLATFORM_NAME="android"
+
+if [ ! -d "$PLATFORM_NAME" ]; then
+    echo "start this script in the source root directory"
+    exit 1
+fi
+
+BUILD_DIR="$PWD/3rdParty/$PLATFORM_NAME"
```

Comment:
This file contains a lot of copy-pasted code from this file: https://github.com/BOINC/boinc/blob/master/android/buildAndroidBOINC-CI.sh
Maybe this is the right place to move common code to some other file and call it from there?

Reply:
I want to think about this.
Maybe import common bash file.

Reply:
I create a common ndk file.